components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,259 @@
+1. Select the **Automate Instances** module.
+1. Select the **New** button.
+1. Enter the following details:
+
+   - **Name**: a unique name for the integration.
+   - **Instance URL**: URL of the Chef Automate instance.
+   - **Automate API token**: an API token generated for Chef Automate with data-feed-service authorization.
+   - **ServiceNow user**: a ServiceNow user name with **ITIL** and `x_chef_automate.api` roles.
+   - **ServiceNow password**: password of the ServiceNow user.
+
+1. Select the **Test Connectivity** button. The application checks that the values specified are correct and the connectivity between Chef Automate and the application establishes.
+
+   A successful test displays the **service-now automate connectivity passed** message. Else, an error message displays detailing the connectivity or credentials issues and help you to resolve.
+
+1. Select **Submit**. The ServiceNow app creates the Chef Automate instance detail.
+
+{{< figure src="/images/automate/snow_integration_configure_sn.png" alt="Chef Automate Instances">}}
+
+### Configure the Chef Automate Integration App from Chef Automate
+
+Follow these steps to configure the Chef Automate Integration App from Chef Automate:
+
+1. Confirm that you have the `x_chef_automate.api` role.
+1. Navigate to **Settings** from the **Chef Automate** menu.
+1. Select the **Data Feeds** link from **Settings**.
+1. Select the **Create Data Feed** button.
+1. Enter the following information:
+
+   - **Name**: a unique name for the Chef Automate Integration App.
+   - **Data Feed URL**: the URL of the ServiceNow application Datafeed API ending with `api/x_chef_automate/asset`.
+   - **Username**: a ServiceNow user with **ITIL** and `x_chef_automate.api` roles.
+   - **Password**: Password of the user.
+
+   You must specify the fully qualified domain name (FQDN) of the ServiceNow instance to configure the **Data Feed URL** in Chef Automate, which has the format: `FQDN/api/x_chef_automate/asset` . For example, <https://venxxx.service-now.com/api/x_chef_automate/asset>.
+
+1. Select the **Test Data Feed** button. The application checks that the values specified are correct and the connectivity between Chef Automate and the application establishes.
+
+   A successful test displays the **service-now automate connectivity passed** message. Else, an error message displays detailing the connectivity or credentials issues and helps you in resolving the error.
+
+1. Select **Create Data Feed**.
+
+{{< figure src="/images/automate/snow_integration_create_data_feed.png" alt="Create Data Feed">}}
+
+### Configure Application Properties
+
+The Chef Automate Integration App has nine configurable **Application Properties**. You must have the `admin` or `x_chef_automate.admin` roles assigned to change the default values in the **Application Properties**.
+
+`x_chef_automate.chef.default.status`
```

Comment:
I think this is better than the table. Although you might consider putting the Default value on a second line, like:

```
`x_chef_automate.chef.default.status`
: Used to set up the status of the service record as **inserted** or **updated**. 
: Default value: `1`.
```

Which looks like:

![Screenshot 2021-07-15 at 14-34-02 Chef Automate Integration App](https://user-images.githubusercontent.com/8138169/125860798-48e57b95-6b65-43cc-adcd-1b0b4a7fb95b.png)

Reply:
I like the default values on the same line and examples should use the space. A fully-composed setting would be:

Name
: Definition words. Another sentence if needed. Type: Boolean, Integer, String. Valid values: `string`, `int`. Default value: `Default`.
: Example:
    ```
    name = False
    ```

Which looks like:
![Screen Shot 2021-07-15 at 4 08 02 PM](https://user-images.githubusercontent.com/4400151/125868841-94d6039c-880f-4666-860a-e952271ca288.png)

Instead of this:
```
Name
: Definition words. Another sentence if needed. Type: Boolean, Integer, String. Valid values: `string`, `int`.
: Default value: `Default`.
: Example:
    ```
    name = False
    ```
```

![Screen Shot 2021-07-15 at 4 06 26 PM](https://user-images.githubusercontent.com/4400151/125868741-a0dd31a5-2596-4fd5-866e-68e1961773a8.png)

Reply:
We're going with definition lists. `Default value:` at the end of the line & `Example` on its own line.